src/RunProcessAsTask.Tests/ProcessExTests.cs

```diff
@@ -1,151 +1,156 @@
 ﻿using System;
 using System.Diagnostics;
+using System.IO;
 using System.Linq;
+using System.Reflection;
+using System.Reflection.Internal;
 using System.Threading;
 using System.Threading.Tasks;
+using RunProcessAsTask;
 using Xunit;
 
 namespace RunProcessAsTask.Tests
 {
-    public static class ProcessExTests
-    {
-        public class RunAsyncTests
+        public static class ProcessExTests

```

Comment:
Your indent settings are different from the project default; I'm not the maintainer @jamesmanning, but if I were, I'd request that you reformat with 4 spaces, not 8.

The reformat causes a huge diff, even though the changes are minor.